src/Kafka/Metrics/components/MetricsLagAlert.tsx

```diff
@@ -0,0 +1,25 @@
+import { VoidFunctionComponent } from "react";
+import { Alert, AlertActionCloseButton, Card } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export type MetricsLagAlertProps = {
+  onClickClose: () => void;
+};
+
+export const MetricsLagAlert: VoidFunctionComponent<MetricsLagAlertProps> = ({
+  onClickClose,
+}) => {
+  const { t } = useTranslation("metrics");
+  return (
+    <Card>
+      <Alert
+        isInline
+        variant="info"
+        title={t("metrics_lag_title")}
+        actionClose={<AlertActionCloseButton onClose={onClickClose} />}
```

Comment:
@jgiardino @riccardo-forina Should the alert message needs to be closed onClick on the close button?

Reply:
The [PoC](https://redhat-developer.github.io/app-services-ui-components/?path=/story/proof-of-concepts-metrics-lag-metrics-lag-alert--lag-message) shows the close icon, so I suppose the answer is yes.